### PR TITLE
Adding enable-nested script

### PR DESCRIPTION
**Pull request by haagha**
enable-nestedhyperv.ps1

### CODE REVIEW

File: map.json [hunk 1]
```diff
@@ -38,5 +38,10 @@
 		"id" : "linux-alar-fki",
 		"path" : "src/linux/linux-alar-fki.sh",
 		"description" : "alar-fki allows to recover a failed VM. Three recover scenarios are possible: fstab, initrd and kernel. NOTE: use option --run-on-repair. Se the docu for more details: https://github.com/Azure/repair-script-library/blob/master/src/linux/common/helpers/alar/README.md"
+	},
+	{
+		"id" : "enable-nested",
```
Comment (swbae31) [inline map.json:43]: Can you change the ID and the file name to follow the existing conventions? 

Something like:

Id: win-enable-nested-hyperV
File name: win-enable-nested-hyperV.ps1
Description: Enables nested hyperV using attached data disk as OS disk.

File: src/windows/enable-nestedhyperv.ps1 [hunk 2]
```diff
@@ -0,0 +1,106 @@
+$scriptStartTime = get-date -f yyyyMMddHHmmss
```
Comment (swbae31) [inline src/windows/enable-nestedhyperv.ps1:1]: Can you read through https://github.com/Azure/repair-script-library/blob/master/doc/adding_new_scripts.md 

and use the existing Logging functions to log messages and return the right status on failure and success?

File: src/windows/win-enable-nested-hyperV.ps1 [hunk 3]
```diff
@@ -0,0 +1,106 @@
+$scriptStartTime = get-date -f yyyyMMddHHmmss
```
Comment (swbae31) [inline src/windows/win-enable-nested-hyperV.ps1:1]: Can you read through https://github.com/Azure/repair-script-library/blob/master/doc/adding_new_scripts.md

and use the existing Logging functions to log messages and return the right status on failure and success? You would need to import the helper and init scripts

Comment (haagha) [reply src/windows/win-enable-nested-hyperV.ps1]: Made the changes, thanks.

**Pull request merged**